queries.sql

```diff
@@ -15,4 +15,48 @@ SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg 
 --(including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/*
+Inside a transaction update the animals table by setting the species column to unspecified.
+ Verify that change was made. Then roll back the change and verify that the species columns 
+ went back to the state before the transaction.
+*/
+--Start the Transactions
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ Required ] I think it would be better if you included all the steps requested in the project requirements, You omitted the step for verifying if indeed the change was made. Please see the example below for more details on this.

## Example code
```SQL
BEGIN TRANSACTION;
UPDATE animals SET species = 'unspecified';
SELECT species FROM animals;  -- Verify that change was made
ROLLBACK;

SELECT species FROM animals;  -- Verify that change was undone
```

## Project requirements
![Screenshot from 2023-08-24 02-29-05](https://github.com/Diegogagan2587/vet-clinic-database/assets/16392046/4361280d-9e58-4b9b-a897-2bb24b67d5aa)

Reply:
Thanks for pointing it out! Will do the changes!